Mathlib/RingTheory/Radical.lean

```diff
@@ -104,4 +106,57 @@ theorem radical_pow_of_prime {a : M} (ha : Prime a) {n : ℕ} (hn : 0 < n) :
   rw [radical_pow a hn]
   exact radical_of_prime ha
 
+
+-- Theorems for commutative rings
+
+-- TODO: This holds for "nontrivial" monoids - do not need ring assumption.
+theorem radical_ne_zero (a : R) : radical a ≠ 0 := by
```

Comment:
As in comment, we don't need ring structure for this; the theorem holds for "nontrivial" monoid. Should I update it by adding the nontriviality assumption, or define another variable $M_1$ for nontrivial monoid, or just leave it as-is?

Reply:
Updated, wonder if this fits into conventions. For example, is there any convention for the order of arguments, like `(a : M) (h: Nontrivial M)` vs `(h: Nontrivial M) (a : M)`?